rpgame/src/main/java/pk/rpgame/model/items/Potion.java

```diff
@@ -19,4 +18,10 @@ public double getHealth() {
     public void use(LivingEntity entity) {
         entity.setHealth(entity.getHealth() + this.health);
     }
+
+    @Override
+    public String toString() {
+        return super.toString()
+                + " [health: " + this.getHealth() + "]";
```

Comment:
i think we should have one convention for this functions, either like here "[health: sth]", or like in armor "\narmor: sth"